lib/link-active.ts

```diff
@@ -63,9 +66,46 @@ export interface LinkActiveOptions {
      });
    }
 
+   subscribeLinks() {
```

Comment:
Managing way too many subscriptions here. Make more use of rx and merge the observables together into a stream you can subscribe to instead of looping through each linkTo.